content/300-accelerate/650-troubleshoot.mdx

```diff
@@ -86,7 +85,7 @@ Additionally, direct connections could have a significant impact on your databas
 If your application's runtime environment supports Prisma ORM natively and you're considering this strategy to circumvent P6009 and P6004 errors, you might create two `PrismaClient` instances:
 
 1. An instance using the Accelerate connection string (prefixed with `prisma://`) for general operations.
-2. Another instance with the direct database connection string (e.g., prefixed with `postgres://`, `mysql://`, etc.) for specific operations anticipated to exceed 10 seconds in execution time or to result in responses larger than 5MB.
+2. Another instance with the direct database connection string (e.g., prefixed with `postgres://`, `mysql://`, etc.) for specific operations anticipated to exceed 10 seconds in execution time or to result in responses larger than [the configured query response size limit](/accelerate/limitations#response-size-limit).
```

Comment:
We should add [the configured query timeout limit](/accelerate/limitations#query-timeout-limit) instead of hard coded 10 seconds here as well.

Reply:
@ankur-arch

Reply:
> We should add [the configured query timeout limit](/accelerate/limitations#query-timeout-limit) instead of hard coded 10 seconds here as well.

```suggestion
2. Another instance with the direct database connection string (e.g., prefixed with `postgres://`, `mysql://`, etc.) for specific operations anticipated to exceed 10 seconds in execution time or to result in responses larger than [the configured query response size limit](/accelerate/limitations#response-size-limit).
```